packages/tocco-ui/src/Panel/PanelGroup.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import StyledPanelGroup from './StyledPanelGroup'
+
+class PanelGroup extends React.Component {
+  state = {
+    openPanelId: this.props.openPanelId >= 0 ? this.props.openPanelId : -1
```

Comment:
I'd personally prefer `undefined` or `null` over `-1`. IMO `-1` is more of a workaround for non nullable variables or types.